.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--- Provide a general summary of your changes in the title. -->
+
+## Description
+<!--- Use this section to describe your changes. If your changes required
+testing, include information about the testing environment and the tests you
+ran. If your test fixes a bug reported in an issue, don't forget to include the
+issue number. If your PR is still a work in progress, that's totally fine – just
+include a note to let us know. -->
+
+### Types of change
+<!-- What type of change does your PR cover? Is it a bug fix, an enhancement
+or new feature, or a change to the documentation? -->
+
+## Checklist
+<!--- Before you submit the PR, go over this checklist and make sure you can
+tick off all the boxes. [] -> [x] -->
+- [ ] I confirm that I have the right to submit this contribution under the project's MIT license.
+- [ ] I ran the tests, and all new and existing tests passed.
+- [ ] My changes don't require a change to the documentation, or if they do, I've added all required information.
```

Comment:
Is the "documentation" question relevant? Do you mean "readme"?

Reply:
Yeah, readme fits better. Updated in https://github.com/explosion/wikid/pull/21/commits/b6f404258417194b3aab2056af84950b79c24cbf.